java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture5/tasks/_1/Number1.java

```diff
@@ -0,0 +1,84 @@
+package com.zoolatech.lecture5.tasks._1;
+
+/*
+Create a method that accepts an integer N and a variable amount of other integer numbers.
+The method needs to return the value of the Nth (1-based) smallest element in the array.
+Return -1 if the amount of passed integers is less than N.
+Input: 4, 1, 5, 2, 3, 7
+Output: 5 (the 4th smallest number in the array 1, 5, 2, 3, 7 is 5: 1 2 3 5 7)
+Input: 3, 1, 2
+Output: -1
+*/
+
+import java.util.*;
+
+public class Number1 {
+    public static void main(String[] args) {
+
+        ArrayList<Integer> arrayList = new ArrayList<>();
+        arrayList.add(1);
+        arrayList.add(5);
+        arrayList.add(2);
+        arrayList.add(3);
+        arrayList.add(7);
+
+        int N = 4;
+
+        Number1 number1 = new Number1();
+        number1.task(N, arrayList);
+    }
+
+    public void task(int N, ArrayList<Integer> arrayList) {
+        ArrayList<Integer> arrayListOriginal = new ArrayList<>();
+        for (int i = 0; i <= N; i++) {
+            arrayListOriginal.add(i);
```

Comment:
why do we need to add some elements to the `arrayListOriginal`? they will be still replaced during the `Collections.copy` action